src/PresentationalComponents/Cards/OverviewDashbarCard/components/DashbarCardTagOrIcon/TagLabelWithTooltip.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import { Tooltip } from '@patternfly/react-core';
+import { InsightsLabel } from '@redhat-cloud-services/frontend-components/InsightsLabel';
+import { TOTAL_RISK_LABEL_LOWER } from '../../../../../AppConstants';
+
+export const TagLabelWithTooltip = ({ typeOfTag }) => {
+  return (
```

Comment:
In cases like this, where the component does nothing special before the return, you can omit it.


```js
export const TagLabelWithTooltip = ({ typeOfTag }) => (
     <Tooltip
       key={`${typeOfTag}-tooltip`}
       position="bottom"
       content={
         <>
           The total risk of this remediation is
           <strong> {TOTAL_RISK_LABEL_LOWER[typeOfTag]}</strong>, based on the
           combination of likelihood and impact to remediate.
         </>
       }
     >
       <InsightsLabel value={typeOfTag} isCompact />
     </Tooltip>
);
```

Reply:
I have added it now 👍🏼 
Thank you !